dist/obs-bundled-gems.spec

```diff
@@ -63,7 +63,7 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 This package bundles all the gems required by the Open Build Service
 to make it easier to deploy the obs-server package.
 
-%define rake_version 13.0.3
+%define rake_version 13.0.6
```

Comment:
I realized that on top of this change, we need to update this package https://build.opensuse.org/package/show/OBS:Server:Unstable/rubygem-rake

It's a matter of linking to the updated version of that package from https://build.opensuse.org/package/show/devel:languages:ruby:extensions/rubygem-rake

Reply:
done